app/controllers/registrations_controller.rb

```diff
@@ -3,7 +3,8 @@ class RegistrationsController < Devise::RegistrationsController
   private
 
   def sign_up_params
-    params.require(:user).permit(:name, :email, :password, :password_confirmation)
+    password = SecureRandom.hex(10)
```

Comment:
Sounds good, just think of a way of eliminating the need for `SecureRandom.hex(10)`.